x/axelarcork/keeper/query_server.go

```diff
@@ -38,7 +38,7 @@ func (k Keeper) QueryCellarIDs(c context.Context, req *types.QueryCellarIDsReque
 	k.IterateChainConfigurations(ctx, func(config types.ChainConfiguration) (stop bool) {
 		set := types.CellarIDSet{ChainId: config.Id, Ids: []string{}}
 		for _, id := range k.GetCellarIDs(ctx, config.Id) {
-			set.Ids = append(set.Ids, id.Hex())
+			set.Ids = append(set.Ids, id.String())
```

Comment:
curious what the reason for all of these changes was

Reply:
Our usage of `common.Address` was a mix of `Hex()` and `String()`. `String()` just internally calls `Hex()`, and it seemed clearer, so I switched all our usages (at least outside the integration tests) to use the same function.